l10n_it_delivery_note_inter_warehouse/models/stock_delivery_note.py

```diff
@@ -0,0 +1,40 @@
+# Copyright 2023 Ooops404
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl).
+
+from odoo import api, models
+
+
+class StockDeliveryNote(models.Model):
+    _inherit = "stock.delivery.note"
+
+    @api.model
+    def _get_sync_fields(self):
+        return [
+            "date",
+            "transport_datetime",
+            "transport_condition_id",
+            "goods_appearance_id",
+            "transport_reason_id",
+            "transport_method_id",
+            "gross_weight",
+            "net_weight",
+            "packages",
+            "volume",
+        ]
```

Comment:
**issue**: you've added this same method with the same fields both here and in `l10n_it_delivery_not_inter_company`.

It would be better to add it once, maybe to `l10n_it_delivery_note`, so it's not duplicated. With a comment explaining that it's used as a hook for other modules (and that's why it's not used immediately in the module where it's declared)

Reply:
Totally good point! 

Done and cherry picked in both PRs 97ef479d5a27859633ffe8f495f476c0d2155a67

Reply:
When one PR is merged, you will have to remove it from the other one.